src/metrics/metrics_base.nit

```diff
@@ -211,6 +211,19 @@ interface Metric
 
 	# The set of element above the threshold
 	fun above_threshold: Set[ELM] is abstract
+
+	# Sort the metric keys by values
+	fun sort: Array[ELM] do
+		return values.keys_sorted_by_values(new DescMetricComparator)
+	end
+end
+
+private class DescMetricComparator
+	super Comparator
+
+	redef type COMPARED: Comparable
+
+	redef fun compare(a, b) do return b <=> a
```

Comment:
this is the `reverse_default_comparator` ? Maybe just introduce it in lib/standard/collection/sorter.nit